Livecheckables/x264.rb

```diff
@@ -4,6 +4,6 @@ class X264
   # the version information at the time of writing.
   livecheck do
     url "https://artifacts.videolan.org/x264/release-macos/"
-    regex(/href=.+?x264-(r\d+)-/i)
+    regex(%r{href=.*?x264[._-](r\d+)[._-][\da-z]+/?[ '">]}i)
```

Comment:
```suggestion
    regex(%r{href=.*?x264[._-](r\d+)[._-][a-z0-9]+/?["' >]}i)
```

`[a-z0-9]` may be a bit more readable/clear than `[\da-z]` within the context of the character class. As above, I've updated this to use `["' >]` (instead of the differing order).

Reply:
There seem to be a few Livecheckables using `[\da-z]`, shall I add that to the cleanup list?

Reply:
Sure, there only appear to be a few, so it wouldn't take long.